subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -98,7 +103,7 @@ public void buildFinished(BuildResult result) {
     }
 
     @Override
-    public void execute(Throwable failure) {
+    public void execute(@Nullable Throwable failure) {
```

Comment:
❌ This isn't right. This method will throw if given a `null` failure when printing the stack trace in `renderSingleBuildException` -> `constructFailureDetails` -> `FailureDetails#renderStackTrace` -> `AbstractStyledTextOutput#exception`.

Reply:
Whoops, I meant to add this as `NotNull` to clean up the warning and just auto-corrected to the wrong annotation.